.github/workflows/CI.yml

```
@@ -10,10 +10,10 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v1
```

Can probably switch to the newest version

```suggestion
      - uses: julia-actions/setup-julia@v2
```